blog/categories.tt

```diff
@@ -7,51 +7,51 @@
       <h2>Events</h2>
```

Comment:
I'm a bit confused. I don't remember how this gets updated. But I assume it shouldn't be committed here.

I also assume this should be emptied (out of scope of this PR) since it gets updated automatically from external sources.

Reply:
See under `scripts/`.

Confirmed, don't commit any of this please :).